src/encoding/integer/mod.rs

```diff
@@ -39,6 +39,7 @@ use crate::{
 
 use super::PrimitiveValueDecoder;
 
+#[allow(unused)]
```

Comment:
regarding fixing clippy warning, do you want me to remove the mode `rle_v1` too? @progval